Docs/Debugging/Debugging with Eclipse and pyOCD.md

```diff
@@ -0,0 +1,68 @@
+#Debugging with Eclipse and pyOCD
+
+This article explains how to debug an exported mbed project with Eclipse and pyOCD. To debug a project, you need to first export it from mbed as a makefile project, import the makefile project into Eclipse and finally setup debugger configurations. Below are these steps but in much more detail.
+
+##Initial Setup
+1. Download the latest version of eclipse with CDT.
+1. Install the GNU ARM Eclipse plugin.
+	1. Open Eclipse.
+	1. Click the **Help** menu item and select **Install New Software**.
+	1. In the **Work with** box paste the install address and press enter - [http://redmine.laoslaser.org/boards/3/topics/635](http://sourceforge.net/projects/gnuarmeclipse/files/Eclipse/updates/).
+  1. The package **GNU ARM C/C++ Cross Development Tools** should appear. Select it.
+  1. Click **Next** repeatedly and accept licence agreements.
+  1. Click **Finish**. If prompted to restart Eclipse, select **Yes**.
+1. [Install tools necessary for makefile projects](https://developer.mbed.org/handbook/Exporting-to-Make):
```

Comment:
Exporting should all link to https://docs.mbed.com/docs/mbed-os-handbook/en/5.3/dev_tools/third_party/